datasource/ethereum/src/ethereum_adapter.rs

```diff
@@ -19,16 +19,25 @@ pub struct EthereumAdapter<T: web3::Transport> {
 }
 
 lazy_static! {
-    static ref TRACE_STREAM_STEP_SIZE: u64 = ::std::env::var("ETHEREUM_TRACE_STREAM_STEP_SIZE")
+    static ref TRACE_STREAM_STEP_SIZE: u64 = std::env::var("ETHEREUM_TRACE_STREAM_STEP_SIZE")
         .unwrap_or("200".into())
         .parse::<u64>()
         .expect("invalid trace stream step size");
 
     /// Maximum number of chunks to request in parallel when streaming logs.
-    static ref LOG_STREAM_PARALLEL_CHUNKS: u64 = ::std::env::var("ETHEREUM_PARALLEL_BLOCK_RANGES")
+    static ref LOG_STREAM_PARALLEL_CHUNKS: u64 = std::env::var("ETHEREUM_PARALLEL_BLOCK_RANGES")
         .unwrap_or("100".into())
         .parse::<u64>()
         .expect("invalid number of parallel Ethereum block ranges to scan");
+
+    /// Maximum range size for `eth.getLogs` requests that dont filter on
+    /// contract address, only event signature, and are therefore expensive.
+    ///
+    /// According to Ethereum node operators, size 500 is reasonable here.
+    static ref MAX_EVENT_ONLY_RANGE: u64 = std::env::var("GRAPH_ETHEREUM_MAX_EVENT_ONLY_RANGE")
```

Comment:
Why start prefixing with `GRAPH_`? Should we standardize on this (eventually)?

Reply:
Many of our env vars have that prefix, some don't. I like prefixing the variable with something that identifies the application.

Reply:
I'm ok with it. We've been a little inconsistent on this front so far, but should standardize on `GRAPH_` going forward.